VogtObserver3/Test.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtObserver3
+{
+    class Test
+    {
+        public static void Run()
```

Comment:
Ooooh you moved this to its own function! So my previous comment where I said to move RemoveCustomersFromStore, AddCustomerToStore to `Program.cs`... you'd want to move them here.

This is a good practice though. I probably wouldn't name the class Test, but it's good to move the application specific code out of Program.cs.

My program.cs nowadays just setup things like making it so that the program will run as a windows service. All of the application code? Starts in another class, like youve done here.